docs/tutorials/galga.md

```diff
@@ -274,7 +274,7 @@ game.onUpdateInterval(500, function () {
 
 ## Step 10
 
-In the ``||math:pick random||`` set the first value as `8` and the second value as `112`. Finally, click on the first ``||variables:mySprite||`` variable, select ``Rename variable...``, and rename it to ``bogey``. Did you notice that all of the ``||variables:mySprite||`` variables turned into ``||variables:bogey||``?
+In the ``||math:pick random||`` block, set the first value as `8` and the second value as `112`.
 
 ```blocks
 let bogey: Sprite = null
```

Comment:
You'll need to change ``bogey`` back to ``mySprite`` in the last set of blocks if you don't like that name.

Reply:
Argh ... good catch, Galen! I forgot to review the inline code, and I had already changed the variable name to `enemyPlane` before I got to Step 10. I like `bogey`, though. I think I'll change it back. :smile: